specification/purview/data-plane/Azure.Analytics.Purview.Scanning/preview/2021-10-01-preview/scanningService.json

```diff
@@ -1872,63 +3553,115 @@
         "properties": {
           "allOf": [
             {
-              "$ref": "#/definitions/AzureKeyVaultProperties"
+              "$ref": "#/definitions/FilterProperties"
             }
           ]
         }
       }
     },
-    "ErrorModel": {
+    "IntegrationRuntime": {
+      "required": [
+        "kind"
+      ],
       "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
       "properties": {
-        "code": {
+        "kind": {
+          "enum": [
+            "SelfHosted",
+            "Managed"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "IntegrationRuntimeType",
+            "modelAsString": true
+          }
+        }
+      },
+      "discriminator": "kind"
+    },
+    "PatchIntegrationRuntimeRequest": {
+      "type": "object",
+      "properties": {
+        "autoUpdate": {
+          "type": "string"
```

Comment:
Without description, I won't know what `autoUpdate` suppose to be. Sound like a true or false.

Should it be an enum?

Reply:
changed this to enum, but this is not part of the revised commit anymore